samples/.gitignore

```diff
@@ -0,0 +1,2 @@
+avatar-*.svg
+avatars/*.svg
```

Comment:
Do we need this second line? Would be nice if we had some samples that had already downloaded an avatar, that would be nice for regression testing reasons.